moped-editor/src/views/staff/StaffForm.js

```diff
@@ -87,17 +87,15 @@ const StaffForm = ({
   const {
     register,
     handleSubmit,
-    errors,
     control,
     watch,
-    formState: { isSubmitting },
+    formState: { isSubmitting, errors },
     reset,
   } = useForm({
     defaultValues: {
       ...initialFormValues,
       // Roles are stored as an array in the DB but we need to feed the form a string
       roles: findHighestRole(initialFormValues.roles),
-      workgroup_id: initialFormValues?.moped_workgroup?.workgroup_id ?? "",
```

Comment:
Workgroup was recently normalized so we no longer need to get the nested id here with special handling. We get it from the `workgroup_id` column of the `moped_users` table. That comes in `initialFormValues` that is spread a few lines above this.